shared/message/input/Sensors.proto

```diff
@@ -112,7 +112,8 @@ message Sensors {
     repeated Servo  servo  = 6;
     repeated Button button = 7;
     repeated LED    led    = 8;
-    /// Voltage for the whole robot, from the main battery
+    /// Present (input) voltage. This value is 10 times larger than the actual voltage.
+    /// For example, when 10V is supplied, the data value is 100 (0x64)
```

Comment:
Nope, it's just volts
Everything is just SI units.
Just curious why did you think it was 100?

Reply:
This comment quotes the data sheet. It was written before I found out about the `convert` class/utility, when I thought we just had raw values from the CM740 being passed around. After I found out about `convert`, I fixed that PR (I think it was the mostly on the one for `platform` messages), removing these incorrect comments.

This shouldn't have even been on this PR. I don't know how it snuck in lol.